app/locales/cs/overlap.json

```diff
@@ -0,0 +1,10 @@
+{
+  "overlap_title": "Kontrola překrytí",
+  "overlap_para_1": "Zelené kruhy jsou shluky vašich míst, kliknutím je zvětšíte. Červené značky jsou vaše zaznamenané pozice.\n\nStisknutím tlačítka stáhnete veřejná data o potvrzených případech, které jsou znázorněny jako slabé fialové kruhy.",
```

Comment:
Does this line translate to:
"overlap_para_1": "The green trail represents your location history\n\nThe light purple circles represent the public dataset",?
I think it is still the line from before update. Can you update this translation?